src/pages/App.tsx

```diff
@@ -89,6 +91,16 @@ export default function App() {
     }, 1000)
   }, [])
 
+  const stacklyPopup = useStacklyPopup()
+  const [closedStacklyPopup] = useUpdateClosedStacklyPopup()
+  const showStacklyPopup = process.env.REACT_APP_SHOW_STACKLY_POPUP === 'true'
```

Comment:
Move this above the component. I don't want to reinitialise this variable.
```suggestion
  const showStacklyPopup = process.env.REACT_APP_SHOW_STACKLY_POPUP === 'true'
```

Reply:
Good tip, forgot about that. Thanks!